editor/inspector/assets/render-pipeline.js

```diff
@@ -46,15 +46,21 @@ exports.methods = {
     },
 
     updateInterface() {
+        this.updateReadonly(this.pipeline);
         this.$.container.render(this.pipeline);
-        this.updateReadonly(this.$.container);
     },
 
-    updateReadonly(element) {
+    updateReadonly(obj) {
         if (this.asset.readonly) {
-            element.setAttribute('disabled', true);
-        } else {
-            element.removeAttribute('disabled');
+            if ('readonly' in obj && 'value' in obj) {
```

Comment:
in builtin-deferred.rpp sometimes obj is undefined, when key is 'uuid'
`if (obj && 'readonly' in obj && 'value' in obj)`

Reply:
@arsen2010